test/e2e/performanceprofile/functests/utils/label/label.go

```diff
@@ -39,6 +39,22 @@ const (
 	// PerformanceProfileCreator should be added in tests that are validating/verifying performance-profile-creator tool
 	// functionally.
 	PerformanceProfileCreator Feature = "performance-profile-creator"
+
+	// MemoryManager should be added in tests that are meant to test memory manager tests
+	MemoryManager Feature = "memorymanager"
+
+	// Offline should be added in tests that are meant to test offline cpus tests
+	Offline Feature = "offlinecpus"
```

Comment:
lets keep consistency in the naming,
all the other names have dashes between each word so let keep it:
use `memory-manager` instead of `memomrymanager`

Reply:
Addressed it in latest commit